crates/db_schema/src/source/local_user.rs

```diff
@@ -36,6 +36,7 @@ pub struct LocalUser {
   pub show_avatars: bool,
   pub send_notifications_to_email: bool,
   /// Whether to show comment / post scores.
+  // TODO now that there is a vote_display_mode, this can be gotten rid of in future releases.
   pub show_scores: bool,
```

Comment:
You can open a draft PR to remove it, then we merge whenever we start doing breaking changes for 0.20.

Reply:
K I'll make a PR for that in a min.